src/quo2/components/settings/category/component_spec.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo2.components.settings.category.component-spec
```

Comment:
you need to add this test to `quo2.core_spec` or it won't run. did you verify the tests are actually working too? We should be manually checking that tests fail if we swap the expression to ensure we have written them correctly 👍